src/css/home.scss

```diff
@@ -476,3 +476,78 @@ $maxwidth: 1280px;
     }
   }
 }
+
+.ocw-news {
+  max-width: $maxwidth;
+
+  h3 {
+    font-family: Helvetica, sans-serif;
+    font-weight: 300;
+    font-size: 28px;
+    margin-bottom: 20px;
+
+    .ocw {
+      color: $orange;
+      font-weight: 600;
+    }
+  }
+
+  .items-row {
+    height: 375px;
+    display: grid;
```

Comment:
I believe that `display: grid` is only partially supported in ie11, so I'm not sure if we can use it (I have held off on using it so far for this reason)

Reply:
it stinks I know :(

Reply:
although maybe we can try out the PR deploy in an ie11 vm and see if it looks ok

Reply:
@pdpinch just to confirm, is IE11 one of the browsers we need to support?